nocasedict/_nocasedict.py

```diff
@@ -50,7 +50,7 @@
 __all__ = ['NocaseDict']
 
 
-class NocaseDict(object):
+class NocaseDict(MutableMapping):
```

Comment:
Since MutableMapping is a superclass of dict, does this mean that we cannot use isinstance(..., dict).???

Reply:
Correct. And all recommendations I found on how to determine whether an object is a dict recommended `isinstance(obj, Mapping)`. See here for a discussion: https://stackoverflow.com/questions/25231989

Reply:
We discussed this and agreed to stay with MutableMapping, but document that the isinstance test should be done against Mapping or MutableMapping.